package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "customerio-node",
   "description": "A node client for the Customer.io event API. http://customer.io",
-  "version": "0.6.0",
-  "author": "Customer.io (https://customer.io",
+  "version": "0.7.0",
```

Comment:
Please revert the version change. We have some other PRs that need to be merged in and deployed with this, and we'll determine the correct version.